assets/src/edit-story/output/utils/styles.js

```diff
@@ -67,6 +67,14 @@ function CustomStyles() {
                 }
               }
 
+              @media not all and (min-resolution:.001dpcm) {
```

Comment:
Please add a comment explaining what this Safari-only hack is for and reference the GH issue #.

Reply:
Can we move the comment just above this line? 

```js
`...` 
/* ... */
+ `@media not all and (min-resolution:.001dpcm) {`
```